### PR TITLE
Clarify User Interaction for sUSD Trading

### LINKED ISSUE #2303: UI Refresh: Make it more clear users are trading in sUSD
Add a clear CTA when then user has no sUSD and offer bridge option.

- [ ] Change the market name format from e.g. `ETH-Perp` to `ETH/sUSD`
- [ ] Add a CTA to available margin are when the user has no sUSD available
- [ ] The CTA should open the bridge modal
- [ ] Add the bridge step back to the cross margin onboarding flow when the user has no sUSD

See the images below for design reference

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
- [X] Change the market name format from e.g. ETH-Perp to ETH/sUSD (in market dropdown and positions table)
- [x] Add a CTA to available margin are when the user has no sUSD available
- [x] The CTA should open the bridge modal
- [x] Add the bridge step back to the cross margin onboarding flow when the user has no sUSD

## Related issue
closes #2303 

## How Has This Been Tested?
Case 1. User without sUSD in wallet
- See the CTA and click on the `Get sUSD` button to open the Deposit modal with socket integration

Case 2. User with sUSD in wallet or idle margin in smart margin account
- See the manage balance as usual

## Screenshots (if appropriate):
image
image
image

### CODE REVIEW

**Review by platschi — DISMISSED** [review]

File: sections/futures/Trade/TradeBalance.tsx [hunk 1]
```diff
@@ -1,55 +1,97 @@
...
 
 const TradeBalance: React.FC<TradeBalanceProps> = memo(({ isMobile = false }) => {
+	const { t } = useTranslation();
 	const dispatch = useAppDispatch();
+
 	const idleMargin = useAppSelector(selectIdleMargin);
 	const accountType = useAppSelector(selectFuturesType);
 	const availableIsolatedMargin = useAppSelector(selectAvailableMargin);
 	const withdrawable = useAppSelector(selectWithdrawableMargin);
+	const susdBalance = useAppSelector(selectSusdBalance);
```
Comment (avclarke) [inline sections/futures/Trade/TradeBalance.tsx:37]: I don't think we need this, the check should just be based on `idleMargin` which will include wallet balance, idle margin in markets and smart margin account balance.

**Review by avclarke — CHANGES_REQUESTED** [review]
Looks good just one change to replace susdBalance with idle margin.

File: sections/futures/Trade/TradeBalance.tsx (hunk 1 above)
Comment (LeifuChen) [reply sections/futures/Trade/TradeBalance.tsx]: Thanks for pointing this out. Updated.

**Review by avclarke — DISMISSED** [review]
Nice work!

**Review by platschi — APPROVED** [review]

**Review by platschi — APPROVED** [review]

**Pull request merged**